extensions/attachments/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,3 +1,20 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
I do not think we want license headers in quarkus-extension.yaml files. Not your fault, @ploef , it is a broken config of the license plugin. I have just sent https://github.com/apache/camel-quarkus/pull/394 to fix this.

Reply:
not very sure, each file should contain the license header afaik